### PR TITLE
ttyd: log to syslog

### LINKED ISSUE #10665: ttyd: logging double includes timestamps
Maintainer: @tsl0922 
Environment: ath79, 1907

Description:
Logging double includes timestamps
```
hu Nov 28 12:33:04 2019 daemon.err ttyd[3703]: [2019/11/28 12:33:04:0768] NOTICE: sending SIGHUP (1) to process (group) -3718
Thu Nov 28 12:33:04 2019 daemon.err ttyd[3703]: [2019/11/28 12:33:04:0783] NOTICE: process exited with code 1, pid: 3718
Thu Nov 28 12:33:04 2019 daemon.err ttyd[3703]: [2019/11/28 12:33:04:0865] NOTICE: WS closed from 192.168.253.124 (192.168.253.124), clients: 0
Thu Nov 28 12:33:04 2019 daemon.err ttyd[3819]: [2019/11/28 12:33:04:3599] NOTICE: ttyd 1.5.2-56d73a11f2 (libwebsockets 3.1.0)
Thu Nov 28 12:33:04 2019 daemon.err ttyd[3819]: [2019/11/28 12:33:04:3687] NOTICE: tty configuration:
Thu Nov 28 12:33:04 2019 daemon.err ttyd[3819]: [2019/11/28 12:33:04:3717] NOTICE:   start command: /usr/libexec/login.sh
Thu Nov 28 12:33:04 2019 daemon.err ttyd[3819]: [2019/11/28 12:33:04:3746] NOTICE:   close signal: SIGHUP (1)
Thu Nov 28 12:33:04 2019 daemon.err ttyd[3819]: [2019/11/28 12:33:04:3776] NOTICE:   terminal type: xterm-256color
Thu Nov 28 12:33:04 2019 daemon.err ttyd[3819]: [2019/11/28 12:33:04:3863] NOTICE: Creating Vhost 'default' port 7681, 2 protocols, IPv6 off
```

### PR DESCRIPTION
**Pull request by ysc3839**
Maintainer: @tsl0922
Compile tested: sunxi, snapshot
Run tested: sun8i-h2-plus-orangepi-zero

Description:
Fixes #10665

### CODE REVIEW

File: utils/ttyd/patches/100-remove-log-timestamp.patch [hunk 1]
```diff
@@ -0,0 +1,13 @@
+diff --git a/src/server.c b/src/server.c
+index e1ee717..0b3ea52 100644
+--- a/src/server.c
++++ b/src/server.c
+@@ -418,7 +418,7 @@ main(int argc, char **argv) {
+         return -1;
+     }
+ 
+-    lws_set_log_level(debug_level, NULL);
++    lws_set_log_level(debug_level, lwsl_emit_stderr_notimestamp);
```
Comment (karlp) [inline utils/ttyd/patches/100-remove-log-timestamp.patch]: there's also emit syslog?

Comment (ysc3839) [reply utils/ttyd/patches/100-remove-log-timestamp.patch]: What do you mean?

**Comment by ysc3839** [comment]
There's still other issues such as no tags. Don't merge now.
```
Fri Nov 29 02:17:02 2019 user.notice : ttyd 1.5.2 (libwebsockets 3.1.0)
Fri Nov 29 02:17:02 2019 user.notice : tty configuration:
Fri Nov 29 02:17:02 2019 user.notice :   start command: /usr/libexec/login.sh
Fri Nov 29 02:17:02 2019 user.notice :   close signal: SIGHUP (1)
Fri Nov 29 02:17:02 2019 user.notice :   terminal type: xterm-256color
Fri Nov 29 02:17:02 2019 user.notice : Creating Vhost 'default' port 7681, 2 protocols, IPv6 off
```

**Comment by ysc3839** [comment]
Now it works.
```
Fri Nov 29 02:46:29 2019 daemon.notice ttyd[11697]: ttyd 1.5.2 (libwebsockets 3.1.0)
Fri Nov 29 02:46:29 2019 daemon.notice ttyd[11697]: tty configuration:
Fri Nov 29 02:46:29 2019 daemon.notice ttyd[11697]:   start command: /usr/libexec/login.sh
Fri Nov 29 02:46:29 2019 daemon.notice ttyd[11697]:   close signal: SIGHUP (1)
Fri Nov 29 02:46:29 2019 daemon.notice ttyd[11697]:   terminal type: xterm-256color
Fri Nov 29 02:46:29 2019 daemon.notice ttyd[11697]: Creating Vhost 'default' port 7681, 2 protocols, IPv6 off
```

**Pull request merged**